tasks/move-granules/schemas/config.json

```diff
@@ -37,12 +37,6 @@
       "type": "string",
       "description": "The api distribution endpoint"
     },
```

Comment:
Why did you remove this from the schema instead of renaming?

Reply:
cmrGranuleUrlType is not used in `move-granules`, the update metadata functionality has been moved to `update-granules-cmr-metadata-file-links`, do we still need to keep it?

Reply:
No you're right, I was completely spacing that this is a schema for the task.